doc/tutorials/app/intelperc.markdown

```diff
@@ -24,13 +24,13 @@ In order to use depth sensor with OpenCV you should do the following preliminary
 VideoCapture can retrieve the following data:
 
 -#  data given from depth generator:
-    -   CAP_INTELPERC_DEPTH_MAP - each pixel is a 16-bit integer. The value indicates the
-            distance from an object to the camera's XY plane or the Cartesian depth. (CV_16UC1)
+    -   CAP_INTELPERC_DEPTH_MAP - each pixel is a 16-bit or 8-bit integer. The value indicates the
+            distance from an object to the camera's XY plane or the Cartesian depth. (CV_16UC1 or CV_8UC1)
```

Comment:
How the following code works this way?

https://github.com/opencv/opencv/blob/fc1a15626226609babd128e043cf7c4e32f567ca/modules/videoio/src/cap_librealsense.cpp#L199-L202

Reply:
Hi, @dkurt 

To work well, I had to make this code in D435:
    Mat depthMap;
    Mat image;
    Mat irImage;
    Mat gray;

    capture.grab();
    capture.retrieve( depthMap, CAP_INTELPERC_DEPTH_MAP );
    capture.retrieve(    image, CAP_INTELPERC_IMAGE );
    capture.retrieve(  irImage, CAP_INTELPERC_IR_MAP);

    depthMap.convertTo(gray,CV_8U);
    
    imshow("Live 1", gray);
    imshow("Live 2", image);
    imshow("Live 3", irImage);

Reply:
![image](https://user-images.githubusercontent.com/675645/103425612-184dea80-4b92-11eb-8fff-61cdd9a6327e.png)

Mat depthMap;
Mat image;
Mat irImage;
Mat gray;
capture.grab();
capture.retrieve( depthMap, CAP_INTELPERC_DEPTH_MAP );
capture.retrieve(    image, CAP_INTELPERC_IMAGE );
capture.retrieve(  irImage, CAP_INTELPERC_IR_MAP);
imshow("Live 1", depthMap);
imshow("Live 2", image);
imshow("Live 3", irImage);

Reply:
Hi, @dkurt,

Solved see!

Reply:
https://github.com/opencv/opencv/pull/19231/commits/96bd31fa9b7df3e89e0683840700035779d588d6

Reply:
modules/videoio/src/cap_librealsense.cpp

Reply:
https://github.com/opencv/opencv/blob/96bd31fa9b7df3e89e0683840700035779d588d6/modules/videoio/src/cap_librealsense.cpp#L199-L201

Reply:
> depthMap.convertTo(gray,CV_8U);

Wrong way.
In your proposal you can even declare support for 64F with the same success.

Reply:
ok, thanks